tests/Controllers/Multiplayer/RoomsControllerTest.php

```diff
@@ -88,6 +88,7 @@ public function testStoreRealtime()
         $this->assertNotNull($room);
         $this->assertTrue($room->isRealtime());
         $this->assertSame($type, $room->type);
+        $this->assertSame($token->user->user_id, $room->playlist()->first()->owner_id);
```

Comment:
`getKey()` is preferred to get model's primary key

```suggestion
        $this->assertSame($token->user->getKey(), $room->playlist()->first()->owner_id);
```